app/javascript/src/ChangePassword/components/ChangePassword.jsx

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import React, { useState } from 'react'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import { LoginPage, Form, ActionGroup, Button } from '@patternfly/react-core'
+import { HiddenInputs, FlashMessages } from 'LoginPage'
+import { PasswordInput } from 'ChangePassword'
+import type { FlashMessage } from 'Types'
+
+import 'LoginPage/assets/styles/loginPage.scss'
+import brandImg from 'LoginPage/assets/images/3scale_Logo_Reverse.png'
+import PF4DownstreamBG from 'LoginPage/assets/images/PF4DownstreamBG.svg'
+
+type Props = {
+  lostPasswordToken: ?string,
+  url: string,
+  errors: (?FlashMessage)[]
+}
+
+const ChangePassword = ({lostPasswordToken, url, errors}: Props) => {
+  const [password, setPassword] = useState('')
+  const [isPasswordValid, setIsPasswordValid] = useState(undefined)
```

Comment:
Maybe you could save a few lines managing the state of the field valid inside the `PasswordInput`.
Also, I might be missing where the "passwords don't match" check is computed.